app/subscriber/src/features/my-folders/styled/MyFolders.tsx

```diff
@@ -0,0 +1,79 @@
+import styled from 'styled-components';
+import { Col } from 'tno-core';
+
+export const MyFolders = styled(Col)`
+  .options {
+    box-shadow: 0 0 0.5rem #c7c7c7;
+    opacity: 1;
+    padding-left: 1.5em;
+    padding-right: 1.5em;
+    .option {
+      &:hover {
+        text-decoration: underline;
+        color: ${(props) => props.theme.css.sidebarIconHoverColor};
+        cursor: pointer;
+      }
+    }
+  }
+  .folder-name {
+    height: 1.5em;
+    margin-left: 0.5em;
+  }
+  .create-new {
+    margin-left: auto;
+  }
+  /* table styling */
+  .table {
+    width: 100%;
+    overflow: hidden;
+    .re-name {
+      height: 1.5em;
+      padding: 0;
+      &:focus {
+        box-shadow: none;
+      }
+    }
+    .group {
+      background-color: #f9f9f9 !important;
```

Comment:
Can you check if this is really the only way to do this?

Reply:
On it